src/proto/primitive.proto

```diff
@@ -4,6 +4,35 @@ package TbotsProto;
 
 import "proto/geometry.proto";
 
+enum BallCollisionType
+{
+    ALLOW = 0;
+    AVOID = 1;
+}
+
+enum DribblerMode
+{
+    OFF        = 0;
+    INDEFINITE = 1;
+    MAX_FORCE  = 2;
+}
+
+enum MaxAllowedSpeedMode
+{
+    PHYSICAL_LIMIT = 0;
+    STOP_COMMAND   = 1;
+    TIPTOE         = 2;
```

Comment:
nit: Can we define what `TIPTOE` is? The one definition I was able to find is being removed from `primitive_types.h`.

Reply:
done